report_py3o/models/py3o_report.py

```diff
@@ -265,18 +267,23 @@ def _convert_single_report(self, result_path, model_instance, data):
     def _convert_single_report_cmd(self, result_path, model_instance, data):
         """Return a command list suitable for use in subprocess.call"""
         lo_bin = self.ir_actions_report_id.lo_bin_path
+        tmp_user_installation = tempfile.TemporaryDirectory().name
         if not lo_bin:
             raise RuntimeError(
                 _("Libreoffice runtime not available. "
                   "Please contact your administrator.")
             )
-        return [
-            lo_bin,
-            '--headless',
-            '--convert-to',
-            self.ir_actions_report_id.py3o_filetype,
-            result_path,
-        ]
+        return (
+            [
+                lo_bin,
+                '-env:UserInstallation=file:%s' % tmp_user_installation,
+                '--headless',
+                '--convert-to',
+                self.ir_actions_report_id.py3o_filetype,
+                result_path,
+            ],
+            tmp_user_installation,
```

Comment:
Why do we need to return tmp_user_installation to the caller? Can't we clean it here?

Reply:
no, we need this to exist for the actual call to libreoffice to succeed. But I'm also concerned about breaking the API here, I'd prefer to see the the temporary directory passed in an optional keyword argument. If passed, we add the parameter to the command, and the caller is responsible for cleaning up.

Reply:
right, I skimmed on it too quickly. So yes, better doing a `with TemporaryDirectory()` in the caller and pass the temp dir to `_convert_single_report`.

Reply:
I pushed a new version to not break the API